src/auth/auth.ts

```diff
@@ -187,13 +181,6 @@ export const loadAuthToken = async (
       refreshTokenExpiresAt: getTimestampMetricLabel(currentRefreshTokenMetadata.expiresAt),
       jwtExpiresAt: getTimestampMetricLabel(jwtExpiresAt),
     });
-  } else if (loadedAuthTokenState.status === 'expired') {
-    Ajax().Metrics.captureEvent(Events.user.authToken.load.expired, {
-      ...getOldAuthTokenLabels(oldAuthTokenMetadata),
-      refreshTokenId: oldRefreshTokenMetadata.id,
-      refreshTokenCreatedAt: getTimestampMetricLabel(oldRefreshTokenMetadata.createdAt),
-      refreshTokenExpiresAt: getTimestampMetricLabel(oldRefreshTokenMetadata.expiresAt),
-    });
   } else {
```

Comment:
We should probably make this else if (loadedAuthTokenState.status === 'error') otherwise i think that we will end up logging expired tokens as errors anyways.